.kokoro/build.sh

```diff
@@ -86,8 +86,8 @@ graalvm)
     RETURN_CODE=$?
     ;;
 graalvm17)
-    # Run Unit and Integration Tests with Native Image. Skip Arrow tests until https://github.com/googleapis/java-bigquery/issues/2060 is fixed.
-    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test "-Dtest=!com.google.cloud.bigquery.it.ITBigQueryTest#testBQResultSetPaginationSlowQuery+testReadAPIConnectionMultiClose+testReadAPIIterationAndOrder, !com.google.cloud.bigquery.it.ITNightlyBigQueryTest#testIterateAndOrder+testMultipleRuns+testIterateAndOrderDefaultConnSettings, IT*, *ClientTest"
+    # Run Unit and Integration Tests with Native Image
+    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
```

Comment:
Let's use the maven command used to run integration check(line 64) to run both graalVM checks too. Otherwise it will run the nightly checks as well.

Reply:
@mpeddada1 +1 We may want to avoid triggering ITNightly during GraalVM presubmits (it would otherwise slow it down)

Reply:
Good point! Modified the script to exclude nightly tests.

Reply:
@Neenu1995 @prash-mi to make sure that we're not losing test coverage, what is the process of adding these excluded tests as nightly jobs (with native image compilation)?

Reply:
Same way the graalvm presubmbit checks are added, add the cfg files to the nightly folder.

Reply:
Sounds great. I'll do this in a follow-up PR.